org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/DiagnosticsHandler.java

```diff
@@ -24,7 +24,7 @@
 
 public class DiagnosticsHandler extends BaseDiagnosticsHandler {
 	private final ICompilationUnit cu;
-	private final String uri;
+	protected final String uri;

```

Comment:
This change doesn't seem necessary anymore, same for those in CodeActionHandler as well.

Reply:
Fixed.